packages/combo-box/src/vaadin-combo-box-mixin.d.ts

```diff
@@ -145,8 +145,15 @@ export declare class ComboBoxMixinClass<TItem> {
 
   protected readonly _propertyForValue: string;
 
+  protected _dropdownItems: TItem[];
+
   protected _inputElementValue: string | undefined;
 
+  /**
+   * Tag name prefix used by scroller and items.
+   */
+  protected readonly _tagNamePrefix: string;
```

Comment:
This is necessary to detect the tag name used by `scroller` elements. Previously, it was obtained from the dropdown tag name. But now using `this.localName` is not an option because of private extensions that are named differently from their owner, e.g. `vaadin-time-picker-combo-box` should still create `vaadin-time-picker-scroller`.